src/main/java/appeng/datagen/providers/recipes/InscriberRecipes.java

```diff
@@ -160,6 +194,10 @@ public Result(String name) {
             @Override
             public void serializeRecipeData(JsonObject json) {
                 json.addProperty("mode", mode.name().toLowerCase(Locale.ROOT));
+                if (fluixTool) {
+                    output.enchant((output.is(AEItems.FLUIX_SWORD.asItem()) ? Enchantments.SHARPNESS
+                            : Enchantments.BLOCK_FORTUNE), 1);
+                }
```

Comment:
Please don't do this in here. Pass the correctly enchanted output to the recipe result instead (just use a static helper to modify the item as needed)

Reply:
I am still discussing whether it's appropriate to hand out Fortune 1 for the price of 1 fluix crystal or not on discord though.